src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'clear') {
+      for (const prop in state) {
+        if (state.hasOwnProperty(prop)) {
+          delete state[prop];
+        }
```

Comment:
```suggestion

        delete state[prop];

```

Reply:
Done. The solution was updated with suggested changes.